### ISSUE TITLE
Heroku stop allowing this code 

### ISSUE THREAD

**Issue by MrZimp** [developeranaz/URL-TO-MEGA-HEROKU#3]
Heroku, doesn't allow it to be deployed can you find a fix or inform me of a way to deploy it

**Comment by developeranaz** [comment]
actually this won't work,
please evolve to [URL-Magnet-Cloud-Uploader-Heroku](https://github.com/developeranaz/URL-Magnet-Cloud-Uploader-Heroku) 
They both share the same code.
This can upload any url to any cloud including mega

**Comment by MrZimp** [comment]
> actually this won't work,
> please evolve to [URL-Magnet-Cloud-Uploader-Heroku](https://github.com/developeranaz/URL-Magnet-Cloud-Uploader-Heroku) 
> They both share the same code.

Yes I have seen this one I didn't want to use it for the reason, or changing account been way more complex then the other bot, but I'll do with want it have thank for the info.
